runtime/gc_vlhgc/AllocationContextBalanced.cpp

```diff
@@ -37,7 +37,7 @@
 #include "EnvironmentVLHGC.hpp"
 #include "HeapRegionDescriptorVLHGC.hpp"
 #include "HeapRegionManager.hpp"
-#include "MemoryPoolBumpPointer.hpp"
+#include "MemoryPoolAddressOrderedList.hpp"
```

Comment:
if you end up successfully simplifying the code so to deal with base MP, you should remove this include